### ISSUE TITLE
Scale estimates

### ISSUE THREAD

**Issue by parkan** [mediachain/mediachain#15]
It's good to look at existing datasets to help us estimate the scale.

Here's The European Library with 109M records: http://www.theeuropeanlibrary.org/tel4/access/data/opendata

According to this, Flickr receives on the order of 10M photos per day:
https://www.flickr.com/photos/franckmichel/6855169886

LoC: 16M books

CC: [1.1B works](https://stateof.creativecommons.org/2015/)

Spotify catalog: ~30M songs (http://www.nytimes.com/interactive/2015/06/30/business/media/music-streaming-guide.html)

Feel free to add any other large-scale datasets here

For fun, here's the IA petabox storage system: https://archive.org/web/petabox.php

**Comment by denisnazarov** [comment]
The social network numbers are definitely orders of magnitude more than openly licensed, but good to compare with (some sources are better than others().

Instagram: [58,000,000 per day](http://www.statisticbrain.com/instagram-company-statistics/)
Facebook: 300 million per day. (Source: Gizmodo)
Tumblr: [53.3 million](http://expandedramblings.com/index.php/tumblr-user-stats-fact/)

**Comment by bigs** [comment]
imo the CC archive is probably a decent target for now

**Comment by parkan** [comment]
Yeah, the "realistic long-term" scale I'm thinking of is 1-10B based on these figures

**Comment by vyzo** [comment]
So invariably we'll need to shard; let's do that from the beginning.

**Comment by parkan** [comment]
~2M Getty images _on hand right now_

**Comment by parkan** [comment]
Would be good to revisit this on a per-namespace basis, making mental note
